src/components/AddPlaidBankAccount.js

```diff
@@ -26,6 +26,8 @@ const propTypes = {
     /** Contains plaid data */
     plaidData: plaidDataPropTypes,
 
+    selectedPlaidAccountID: PropTypes.string,
```

Comment:
The reviewer checklist says that _all props are defined accurately and each prop has a /** comment above it */_, although it's under _ If a new component is created I verified that:_ so not sure if this should be a blocker or not

Reply:
Good catch, we should probably still do that.